patches/tModLoader/Terraria/ModLoader/GlobalTile.cs

```diff
@@ -319,11 +319,12 @@ public virtual void PostSetupTileMerge()
 	}
 
 	/// <summary>
-	/// This hook runs when any tree is shook. This allows you to add drops when you shake trees.
+	/// <br>This hook runs when any tree is shook. This allows you to add drops when you shake trees.</br>
+	/// <br><see cref="TreeTypes.None"/> will not allow the tree to be shaken.</br>
```

Comment:
`shook` -> `shaken`
Maybe remove the `TreeTypes.None` line since the tense doesn't quite make sense. The documentation on the `CountsAsTreeType` will suffice